scss/modules/datagrid-table-view-light.scss

```diff
@@ -32,12 +32,15 @@
         }
 
         tbody {
-            tr.selected {
-                border-left: none;
-            }
+            tr {
+                &.selected {
+                    border-left: none;
+                }
 
-            .row.even, .row.odd {
-                background: transparent;
+                &:nth-child(2n),
+                &:nth-child(2n+1) {
+                    background: transparent !important;
+                }
```

Comment:
no it isnt but the issue happens only in the automation-tab which i have edited also in the linked PR (it happens because of the same reason as i explained here https://github.com/massiveart/husky/pull/756#discussion_r93019251